src/index.ts

```diff
@@ -64,7 +68,7 @@ export async function createDBConnection() {
 		type: 'postgres',
 		...getConfig().db, // username, password, host, port, database
 		entities: [
-			User, EmailConfirmation, Profile, Event
+			User, EmailConfirmation, Profile, Event, Channel, EventChannel, Message
```

Comment:
Instead of manually specifying all the entities, you could use a glob pattern for the directory instead like:
`${__dirname}/entities/**/*{.js,.ts}`